gluten-core/src/main/scala/io/glutenproject/execution/SortExecTransformer.scala

```diff
@@ -238,7 +238,7 @@ case class SortExecTransformer(sortOrder: Seq[SortOrder],
                            validation: Boolean): RelNode = {
     val args = context.registeredFunction
     val sortFieldList = new util.ArrayList[SortField]()
-    sortOrder.map(order => {
+    sortOrder.foreach(order => {
```

Comment:
In a UT case, a scala `Stream` is used to represent a sort order with three col to sort. Since `Stream` is on demand data structure which means not all the values will be evaluated until you need them. If `map` was used here, only the first element will be evaluated. Another reason for not using  `map` is to avoid unnecessarily create a data structure to maintain the mapping result.